extension/js/background_page/background_page.ts

```diff
@@ -59,8 +59,11 @@ console.info('background_process.js starting');
   BrowserMsg.bgAddListener('ajaxGmailAttachmentGetChunk', BgHandlers.ajaxGmailAttachmentGetChunkHandler);
   BrowserMsg.bgAddListener('settings', BgHandlers.openSettingsPageHandler);
   BrowserMsg.bgAddListener('update_uninstall_url', BgHandlers.updateUninstallUrl);
+  BrowserMsg.bgAddListener('create_alarm_with_delay', BgHandlers.createAlarmWithDelay);
+  BrowserMsg.bgAddListener('clear_alarm', BgHandlers.clearAlarm);
```

Comment:
this file isn't used in manifest v3, such listeners should be added in https://github.com/FlowCrypt/flowcrypt-browser/blob/05ca61543f39dca62997b3554adbba9e1559a182/extension/js/service_worker/background.ts